src/main/kotlin/com/rustyrazorblade/easycassstress/commands/Run.kt

```diff
@@ -434,7 +436,8 @@ class Run(val command: String) : IStressCommand {
 
     private fun createRunners(plugin: Plugin, metrics: Metrics, fieldRegistry: Registry, rateLimiter: RateLimiter?): List<ProfileRunner> {
         val runners = IntRange(0, threads - 1).map {
-            println("Connecting")
+            // println("Connecting")
+            println("Connecting to cassandra cluster ...")
             val context = StressContext(session, this, it, metrics, fieldRegistry, rateLimiter)
```

Comment:
We should already have the Cassandra connection by now, because the session is the result of creating the connection.  I need to double check when I have more time though.